Library/Homebrew/cmd/audit.rb

```diff
@@ -479,6 +479,19 @@ def audit_github_repository
     end
   end
 
+  def audit_tls_potential
+    # This only handles stable URLs at the moment. I'd like to give it some field
+    # testing on a relatively simple check before we expand it further.
+    return unless @online
+
+    url = formula.stable.url
+    unless url =~ %r[^https://]
+      if system "curl", "--connect-timeout", "3", "--silent", "-I", "https://#{url.split("http://")[1]}", "-o", "/dev/null"
```

Comment:
It’s probably better to use `url.sub("http://", "https://")` instead of a `split` here.